src/Application.php

```diff
@@ -320,4 +334,52 @@ private function sortTransformationRows(array $transformationConfig): array
         $transformationConfig['rows'] = $result;
         return $transformationConfig;
     }
+
+    private function isSuitableTransformation(array $row, TransformationV2 $savedTransformation): bool
+    {
+        if (empty($row['configuration']['input'])) {
+            return true;
+        }
+        $rowInputMappings = (array) array_combine(
+            array_map(fn($v) => $v['destination'], $row['configuration']['input']),
+            $row['configuration']['input']
+        );
+
+        foreach ($savedTransformation->getInputMappingTables() as $savedInputMappingTable) {
```

Comment:
Just going to comment to make sure I understand what is happening here 😄 .

So, it checks the row and looks if there is already an IM with that `destination`.  If there is already one there, then it returns false if there is a difference in the filters being applied.

That seems ok to me.  But what about columns?  what if the 2 have different columns selected?

Reply:
@pivnicek 

> So, it checks the row and looks if there is already an IM with that destination. If there is already one there, then it returns false if there is a difference in the filters being applied.

exactly

> That seems ok to me. But what about columns? what if the 2 have different columns selected?

if you have 2 tables with different selected columns, we merges them to one IM, see
https://github.com/keboola/app-transformation-migrate/blob/master/src/ValueObjects/TransformationV2.php#L254L280
but for different e.g. `changeSince` we must create a new transformation

Reply:
aha, rozumim.  diky za vysvetleni